src/client.js

```diff
@@ -9,9 +9,16 @@ import { template, templateStyles } from './app/lib/joinUsTemplate';
 const data = window.SIMORGH_DATA || {};
 const root = document.getElementById('root');
 
-Loadable.preloadReady().then(() => {
-  hydrate(<ClientApp data={data} routes={routes} />, root);
-});
+// Only hydrate the client if we're on a known route
+// When on an unknown route, the SSR will be discarded and the user will only
+// see a blank screen. This should never happen on BBC domains, but web page
+// archival/caching sites such as Google webcache and web.archive.org, among
+// others.
+if (window.SIMORGH_DATA.path === window.location.pathname) {
+  Loadable.preloadReady().then(() => {
+    hydrate(<ClientApp data={data} routes={routes} />, root);
+  });
+}
```

Comment:
Can we add a logger info call to this on an `else` please? If this breaks "Is there a console log" will be an easy debug imo